Cargo.toml

```diff
@@ -31,6 +31,7 @@ env_logger = { version = "0.11.7", default-features = false }
 flate2 = "1.0.30"
 git2 = { version = "0.18", default-features = true, features = ["vendored-openssl"] }
 glob = { version = "0.3.1", default-features = false }
+hex = { version = "0.4.3", default-features = false }
```

Comment:
Instead of adding a new library can we use `sp_core`?
```
use sp_core::bytes::{from_hex, to_hex};
```

Reply:
Absolutely right, I didn't realize of that when exploring the codebase. Done in [130b650](https://github.com/r0gue-io/pop-cli/pull/592/commits/130b650bcb46a12de6e4085d8d6ccf685f37aacf).